hdbt.theme

```diff
@@ -1476,3 +1482,22 @@ function hdbt_preprocess_paragraph__school_search(array &$variables): void {
     $variables['#attached']['drupalSettings']['helfi_school_search']['cookie_privacy_url'] = $privacyUrl->toString();
   }
 }
+
+/**
+ * Implements hook_preprocess_eu_cookie_compliance_popup_info().
+ */
+function hdbt_preprocess_eu_cookie_compliance_popup_info(array &$variables) {
+  $primaryLanguages = [
+    'fi',
+    'en',
+    'sv',
+  ];
+
+  $language = Drupal::languageManager()->getCurrentLanguage();
```

Comment:
I think current language should be defined with parameter these days. It should be either interface lang or content lang depending on context:
`getCurrentLanguage(LanguageInterface::TYPE_INTERFACE)`
or
`getCurrentLanguage(LanguageInterface::TYPE_CONTENT)`

In this case I believe it should be content lang but you know better what's happening in this PR :)

Reply:
Risto has been updating these lately. Check this PR for reference: https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/493/files

Reply:
Good point! I pushed a change to fix this one in the `hdbt_preprocess()`. The commented part of code is already removed so no need fix in that.
I also believe it should be content lang.